# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/moz.build
# Commit: ccfa767dba64
# Full Hash: ccfa767dba644dc193e0246eb8e8ff3377e8b8a5
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-01-09 03:40:18
# Regressor Bug: 1477756
# File Overlap Count: 1
# Description:
#   Bug 1477756 - Client-side bindings mirror for precise CC, and merge similar codepaths. r=handyman
#   
#   * Context loss using RAII
#   * Move Program reflection Client-side
#   
# ==============================================================================

diff -r 71c122ac0ca7 -r ccfa767dba64 dom/canvas/moz.build
--- a/dom/canvas/moz.build	Wed Jan 08 22:19:14 2020 +0000
+++ b/dom/canvas/moz.build	Wed Jan 08 22:19:16 2020 +0000
@@ -63,13 +63,10 @@
     'TextMetrics.h',
     'WebGLChild.h',
     'WebGLCommandQueue.h',
-    'WebGLContextEndpoint.h',
     'WebGLCrossProcessCommandQueue.h',
-    'WebGLErrorQueue.h',
+    'WebGLIpdl.h',
     'WebGLParent.h',
-    'WebGLShaderPrecisionFormat.h',
     'WebGLTypes.h',
-    'WebGLVertexArrayObject.h',
 ]
 
 # Canvas 2D and common sources
@@ -96,22 +93,17 @@
     'HostWebGLContext.cpp',
     'ProducerConsumerQueue.cpp',
     'TexUnpackBlob.cpp',
-    'WebGL1Context.cpp',
     'WebGL2Context.cpp',
     'WebGL2ContextBuffers.cpp',
     'WebGL2ContextFramebuffers.cpp',
     'WebGL2ContextMRTs.cpp',
-    'WebGL2ContextPrograms.cpp',
     'WebGL2ContextQueries.cpp',
     'WebGL2ContextRenderbuffers.cpp',
     'WebGL2ContextSamplers.cpp',
     'WebGL2ContextState.cpp',
     'WebGL2ContextSync.cpp',
-    'WebGL2ContextTextures.cpp',
     'WebGL2ContextTransformFeedback.cpp',
     'WebGL2ContextUniforms.cpp',
-    'WebGL2ContextVAOs.cpp',
-    'WebGLActiveInfo.cpp',
     'WebGLBuffer.cpp',
     'WebGLChild.cpp',
     'WebGLContext.cpp',
@@ -128,8 +120,6 @@
     'WebGLContextVertexArray.cpp',
     'WebGLContextVertices.cpp',
     'WebGLCrossProcessCommandQueue.cpp',
-    'WebGLErrorQueue.cpp',
-    'WebGLExtensionBlendMinMax.cpp',
     'WebGLExtensionColorBufferFloat.cpp',
     'WebGLExtensionColorBufferHalfFloat.cpp',
     'WebGLExtensionCompressedTextureASTC.cpp',
@@ -140,27 +130,19 @@
     'WebGLExtensionCompressedTextureRGTC.cpp',
     'WebGLExtensionCompressedTextureS3TC.cpp',
     'WebGLExtensionCompressedTextureS3TC_SRGB.cpp',
-    'WebGLExtensionDebugRendererInfo.cpp',
-    'WebGLExtensionDebugShaders.cpp',
     'WebGLExtensionDepthTexture.cpp',
     'WebGLExtensionDisjointTimerQuery.cpp',
     'WebGLExtensionDrawBuffers.cpp',
-    'WebGLExtensionElementIndexUint.cpp',
     'WebGLExtensionEXTColorBufferFloat.cpp',
     'WebGLExtensionFragDepth.cpp',
     'WebGLExtensionInstancedArrays.cpp',
-    'WebGLExtensionLoseContext.cpp',
-    'WebGLExtensionMOZDebug.cpp',
     'WebGLExtensions.cpp',
     'WebGLExtensionShaderTextureLod.cpp',
     'WebGLExtensionSRGB.cpp',
-    'WebGLExtensionStandardDerivatives.cpp',
-    'WebGLExtensionTextureFilterAnisotropic.cpp',
     'WebGLExtensionTextureFloat.cpp',
     'WebGLExtensionTextureFloatLinear.cpp',
     'WebGLExtensionTextureHalfFloat.cpp',
     'WebGLExtensionTextureHalfFloatLinear.cpp',
-    'WebGLExtensionVertexArray.cpp',
     'WebGLFormats.cpp',
     'WebGLFramebuffer.cpp',
     'WebGLMemoryTracker.cpp',
@@ -170,19 +152,16 @@
     'WebGLRenderbuffer.cpp',
     'WebGLSampler.cpp',
     'WebGLShader.cpp',
-    'WebGLShaderPrecisionFormat.cpp',
     'WebGLShaderValidator.cpp',
     'WebGLSync.cpp',
     'WebGLTexelConversions.cpp',
     'WebGLTexture.cpp',
     'WebGLTextureUpload.cpp',
     'WebGLTransformFeedback.cpp',
-    'WebGLUniformLocation.cpp',
     'WebGLValidateStrings.cpp',
     'WebGLVertexArray.cpp',
     'WebGLVertexArrayFake.cpp',
     'WebGLVertexArrayGL.cpp',
-    'WebGLVertexArrayObject.cpp',
     'WebGLVertexAttribData.cpp',
 ]
 